core/Symbols.h

```diff
@@ -489,8 +489,8 @@ class Symbol final {
     std::string toStringFullName(const GlobalState &gs) const;
     std::string showFullName(const GlobalState &gs) const;
 
-    // Not printed when showing name table
-    bool isHiddenFromPrinting(const GlobalState &gs) const;
+    // Returns true if the symbol or any of its children are not in the name table. False otherwise.
```

Comment:
```suggestion
    // Returns true if the symbol or any of its children are not in the symbol table. False otherwise.
```

This comment is from a time when we called the symbol table the name table despite also having an actual data structure called the name table.